Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Bing Service/BingCode.bind

```diff
@@ -11,4 +11,11 @@ var searchConfig = new BingSearchConfig
     QueryType = BingQueryType.Search
 };
 
+// To perform a single call to get 50 items:
 ListView.ItemsSource = await BingService.Instance.RequestAsync(searchConfig, 50);
+
+// To make an incremental loading of search results:
+var service = BingService.GetIncrementalLoadingService(searchConfig);
+var collection = new IncrementalLoadingCollection<BingService, BingResult>(service, 50);
```

Comment:
Because in this way we can customize the way IncrementalLoadingCollection works, for example by adding actions to be executed when loading starts or finishes. I show this behavior in the Sample for BingService: https://github.com/marcominerva/UWPCommunityToolkit/blob/dev/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Bing%20Service/BingPage.xaml.cs. In this case, I show the progress ring when the incremental loading starts, and hide it at the end of the operation.